docs/testcases/UI/Pipe_CLI/2847_introduce extended attributes support.md

```diff
@@ -0,0 +1,32 @@
+# Introduce extended attributes support in pipe fuse
+
+Test verifies that
+- extended attribute calls are supported
+- by default only attributes with user prefix are processed, all other attributes are ignored.
+
+**Prerequisites**:
+- Admin user
+
+**Preparations**:
+1. Create object storage `storage1`
+2. In the `storage1` create `file1`
+
+| Steps | Actions | Expected results |
+| :---: | --- | --- |
+| 1 | Login as the admin user from the prerequisites | |
+| 2 | Open the Tools page | |
+| 3 | Select `centos` tool | |
+| 4 | Run tool with *Custom settings* | |
```

Comment:
I suggest to change the step to 'Launch the tool on on-demand instance'.

Reply:
Done